sale_disable_inventory_check/models/product_category.py

```diff
@@ -0,0 +1,16 @@
+# Copyright Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    check_stock_on_sale = fields.Boolean(
+        company_dependent=True,
+        help="This checkbox disable warning 'Not enough inventory'"
+             " when there isn't enough product in stock",
```

Comment:
I have to come back to the field semantics. If the field is called 'Check stock on sale', and this option is active, I would expect stock to be checked on the sale order. I don't think that is what is currently the case.

Reply:
Might be as simple as adding `skip_` to the start of the field name.

Reply:
That is a double negation: activate an option so that something is not happening. What do the usability guys think about that?